spec.bs

```diff
@@ -4824,9 +4824,10 @@ frequency capping.
 </dl>
 
 A <dfn>seller capability</dfn> is a permission granted by an [=interest group=] to sellers. It's
-"`interest-group-counts`" or "`latency-stats`". Each value reprensents a single bit in a 4 bytes
-[=byte sequence=]. The 4 bytes [=byte sequence=] can store no more than 32 capabilities, and need to
-add more bytes if more than 32 capabilities are supported.
+"`interest-group-counts`" or "`latency-stats`". Each value represents a single bit in a 4-byte
+

```

Comment:
I think we somehow picked up another line here -- sorry if that happened due to accepting my suggestion.